climada/hazard/tc_clim_change.py

```diff
@@ -1,174 +1,329 @@
+        3.493864286,3.552964286,3.592364286,3.630664286,3.672464286,3.734364286,3.789764286,
+        3.838164286,3.882264286,3.936064286,3.984064286,4.055764286,4.098964286,4.122364286,
+        4.172064286,4.225264286,4.275064286,4.339064286,4.375864286,4.408064286,4.477764286
         ]
 
-    return na + ep + wp + sp + ni + si
+    return gmst_data, gmst_start_year, gmst_end_year, rcps
 
-
-def calc_scale_knutson(ref_year=2050, rcp_scenario=45):
+def get_knutson_data():
     """
-    Comparison 2081-2100 (i.e., late twenty-first century) and 2001-20
-    (i.e., present day). Late twenty-first century effects on intensity and
-    frequency per Saffir-Simpson-category and ocean basin is scaled to target
-    year and target RCP proportional to total radiative forcing of the
-    respective RCP and year.
+    Get data to generate figures 1b to 4b in Knutson et al., (2020):
 
-    Parameters
-    ----------
-    ref_year : int, optional
-        year between 2000 ad 2100. Default: 2050
-    rcp_scenario: int, optional
-        26 for RCP 2.6, 45 for RCP 4.5. The default is 45
-        60 for RCP 6.0 and 85 for RCP 8.5.
-
-    Returns
-    -------
-    factor : float
-        factor to scale Knutson parameters to the give RCP and year
+    Tropical cyclones and climate change assessment. Part II: Projected 
+        response to anthropogenic warming. Bull. Amer. Meteor. Soc., 101 (3), E303–E322, 
+        https://doi.org/10.1175/BAMS-D-18-0194.1.
+    
+    for 4 variables (i.e., cat05 frequency, cat45 frequency, intensity precipitation rate), 
+    6 regions (i.e., N. Atl, NW Pac., NE Pac., N. Ind, S. Ind., SW Pac.) and 
+    5 metrics (i.e., 5% or 10%, 25%, median, 75%, 95% or 90%).
     """
-    # Parameters used in Knutson et al 2015
-    base_knu = np.arange(2001, 2021)
-    end_knu = np.arange(2081, 2101)
-    rcp_knu = 45
-
-    # radiative forcings for each RCP scenario
-    rad_force = pd.read_excel(TOT_RADIATIVE_FORCE)
-    years = np.array([year for year in rad_force.columns if isinstance(year, int)])
-    rad_rcp = np.array([int(float(sce[sce.index('.') - 1:sce.index('.') + 2]) * 10)
-                        for sce in rad_force.Scenario if isinstance(sce, str)])
-
-    # mean values for Knutson values
-    rf_vals = np.argwhere(rad_rcp == rcp_knu).reshape(-1)[0]
-    rf_vals = np.array([rad_force.iloc[rf_vals][year] for year in years])
-    rf_base = np.nanmean(np.interp(base_knu, years, rf_vals))
-    rf_end = np.nanmean(np.interp(end_knu, years, rf_vals))
-
-    # scale factor for ref_year and rcp_scenario
-    rf_vals = np.argwhere(rad_rcp == rcp_scenario).reshape(-1)[0]
-    rf_vals = np.array([rad_force.iloc[rf_vals][year] for year in years])
-    rf_sel = np.interp(ref_year, years, rf_vals)
-    return max((rf_sel - rf_base) / (rf_end - rf_base), 0)
+
+    knutson_data = np.empty((4, 6, 5))
+
+    # fig 1 in Knutson et al. 2020 (except min, max metrics and global region)
```

Comment:
Can you in this and the similar comments below also specify which variable these data are for?